CHANGES.md

```diff
@@ -1,3 +1,17 @@
+## edge-19.11.3
+
+* CLI
+  * Added a check that ensures using `--namespace` and `--all-namespaces`
+    results in an error as they are mutually exclusive
+* Web UI
```

Comment:
Hey @zaharidichev, you can delete the `Web UI` section, this only affected a certain browser version and isn't necessary to call out to users. Thanks 👍